census.go

```diff
@@ -432,6 +434,147 @@ func (v *vocdoniHandler) censusQueueInfo(msg *apirest.APIdata, ctx *httprouter.H
 	return ctx.Send(data, http.StatusOK)
 }
 
+type Token struct {
+	Address    string `json:"address"`
+	Blockchain string `json:"blockchain"`
+}
+
+func (v *vocdoniHandler) censusTokenNFT(msg *apirest.APIdata, ctx *httprouter.HTTPContext) error {
+	req := struct {
```

Comment:
I think it makes more sense to move this type to types.go, keep a slice and use it for both (NFTs and ERC20). Just on the ERC20 case, for now, check the len is 1. But in the future it will probably change.